.github/workflows/snyk-scan.yml

```diff
@@ -0,0 +1,29 @@
+# This is a basic workflow to help you get started with Actions
```

Comment:
Does this file clash with [our own](https://github.com/guardian/PrivateBin/blob/master/.github/workflows/snyk.yml)?

Reply:
Good spot! I have a branch without this file in it but don't have push access to this repo, would you mind adding me? Or my team digital investigations

Reply:
Added https://github.com/orgs/guardian/teams/guardian-developers-write with write permissions now.

Reply:
thank you!